src/semantic-router/pkg/config/config.go

```diff
@@ -261,14 +261,14 @@ func (c *RouterConfig) GetCacheSimilarityThreshold() float32 {
 
 // Category represents a category for routing queries
 type ModelScore struct {
-	Model string  `yaml:"model"`
-	Score float64 `yaml:"score"`
+	Model        string  `yaml:"model"`
```

Comment:
use_reasoning: true|false
    reasoning_description: "Why reasoning is needed"
    reasoning_effort: "low|medium|high"

should we move all these three reasoning related vars into model score?